.github/CODEOWNERS

```diff
@@ -415,7 +415,7 @@
 /core/templates/domain/subtopic_viewer/ @kevintab95
 /core/templates/domain/topic/ @kevintab95
 /core/templates/domain/topic_viewer @kevintab95
-/core/templates/pages/classroom-page/ @kevintab95
+/core/templates/pages/classroom-page/ @kevintab95 @dchen97
```

Comment:
> Adds @dchen97 to codeowners for some user-facing template files.

@seanlip, This will include her to own the .ts, .spec.ts (And other ts files) is that expected? I'm fine with either, just wanted to confirm.